GetCanvasTODOList/index.js

```diff
@@ -2,13 +2,19 @@ const axios = require("axios");
 const main = async(inputs, auths, context) => {
   console.log("Fetching data");
   let { domain, token } = inputs;
-  const {data} = await axios.get(
-    `https://${domain}/api/v1/users/self/todo`,
-    {
-      headers: { 'Authorization': `Bearer ${token}` }
-    }
-  );
-  return data;
+  try{
```

Comment:
Remove try catch, also remove on Dcoder block, then rebuild and release a new version for both blocks.